app/controllers/api/v1/api_controller.rb

```diff
@@ -34,8 +34,18 @@ def log_exception(exception)
         logger.error exception.backtrace.join("\n") # TODO: amir - Revisit this.
       end
 
-      def render_data(data: {}, status: :ok, include: nil, serializer: nil, options: {})
-        render json: data, status:, include:, root: 'data', serializer:, options:
+      def render_data(data: {}, status: :ok, serializer: nil, each_serializer: nil, options: {})
+        args = {
+          json: data,
+          root: 'data',
+          status:,
+          serializer:,
+          options:
+        }
+        # Allow ActiveModelSerializer to guess which Serializer to use unless manually specified
+        args[:each_serializer] = each_serializer if each_serializer.present?
+
+        render args
```

Comment:
I don't understand the difference between `serializer` and `each_serializer`

Doesn't ActiveModelSerializer guess which Serializer to use by default?

When would you use `each_serializer` instead of `serializer`